### PR TITLE
MR-257 Google Analytics

### PR DESCRIPTION
**Pull request by rdgfuentes**

### CODE REVIEW

File: applications/fishing-map/src/features/analysis/Analysis.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useRef, useMemo, useState, Fragment } from 'react'
 import cx from 'classnames'
+import * as ua from 'react-ga'
```
Comment (j8seangel) [inline applications/fishing-map/src/features/analysis/Analysis.tsx:3]: Is there any specific reason to import the entire package ? We might can see some bundle size if importing just what we need.

File: applications/fishing-map/src/features/workspace/heatmaps/HeatmapsSection.tsx [hunk 2]
```diff
@@ -133,3 +165,2129 @@ function HeatmapsSection(): React.ReactElement {
 }
 
 export default HeatmapsSection
+
+// const dv = {
```
Comment (j8seangel) [inline applications/fishing-map/src/features/workspace/heatmaps/HeatmapsSection.tsx:169]: oops 👀

File: applications/fishing-map/src/features/app/analytics.hooks.ts [hunk 3]
```diff
@@ -1,40 +1,49 @@
...
+      //   ua.set({ sendHitTask: null })
+      // }
+    }
+  }, [])
+
+  useEffect(() => {
+    if (userData && GOOGLE_UNIVERSAL_ANALYTICS_ID) {
+      ua.set({
+        user_properties: {
+          user_id: userData.id,
```
Comment (j8seangel) [inline applications/fishing-map/src/features/app/analytics.hooks.ts:36]: I've just realised this is using snake_case here but camelCase above... is this correct?

**Review by j8seangel — COMMENTED** [review]
Just a few comments and something large to delete but in general it LGTM

File: applications/fishing-map/src/features/analysis/Analysis.tsx (hunk 1 above)
Comment (rdgfuentes) [reply applications/fishing-map/src/features/analysis/Analysis.tsx:3]: Ok, I'll change it to import just the methods that are used

File: applications/fishing-map/src/features/app/analytics.hooks.ts (hunk 3 above)
Comment (rdgfuentes) [reply applications/fishing-map/src/features/app/analytics.hooks.ts:36]: Just to continue setting them as they were when we used gtag... I'll change it to camelCase

File: applications/fishing-map/src/features/workspace/heatmaps/HeatmapsSection.tsx (hunk 2 above)
Comment (rdgfuentes) [reply applications/fishing-map/src/features/workspace/heatmaps/HeatmapsSection.tsx]: Fixed

**Pull request merged**